CondFormats/RPCObjects/test/RPCReadOutMapBuilder.cc

```diff
@@ -73,7 +72,7 @@ void RPCReadOutMapBuilder::endJob() {
 void RPCReadOutMapBuilder::beginJob() {
   cout << "BeginJob method " << endl;
   cout << "Building RPC Cabling" << endl;
-  cabling = new RPCReadOutMapping("My map V-TEST");
+  RPCReadOutMapping cabling;
```

Comment:
probably better to keep the original argument as the following?
`RPCReadOutMapping cabling("My map V-TEST");`

Reply:
It's just a test naming, for sure doesn't loose functionality, and when I tried it didnt work immediately. If the owners of the code want to change it they can do it in a follow-up somewhen in the future.